### DISCUSSION TITLE
GTS2e swimming sync with strava

### DISCUSSION THREAD

**Discussion by Zainzzkk** [zepp-health/zeppos-forum#121] [User Feedback]
Hi.  The GTS2e does not properly sync with strava.  For all pool swims, the distance is right, but the time is wrong and only shows as 1second.  It is working fine on GTS 3.

**Comment by Xuemuyang** [comment]
Hi, is it possible to provide a screenshot of the relevant error so that we can easily troubleshoot the problem.

**Reply by Zainzzkk** [reply]
![Screenshot_20230111_213328_Strava.jpg](https://user-images.githubusercontent.com/51957827/228180258-5bdfd9b6-804a-430b-83bf-4d09d6e95f2a.jpg)

![Screenshot_20230111_213431_Zepp.jpg](https://user-images.githubusercontent.com/51957827/228180262-785266ce-02e0-44a1-894e-67c64772ee3d.jpg)

**Reply by MyoungXUE** [reply]
Thank you for your cooperation, please wait for feedback from relevant colleagues.

**Reply by MyoungXUE** [reply]
Hello, in order to facilitate our troubleshooting, we need to provide the user ID again, because the user ID is more private, we suggest contacting us by email, my email is xuemuyang@zepp.com

Click on your avatar in the Zepp App to see the ID.
